### ISSUE TITLE
Comments immediately following an interpolation are not recognized as comments

### ISSUE THREAD

**Issue by andrew-skybound** [sass/sass#1894]
Not sure if this is defined behavior or not but it struck me as odd—especially since comments _are_ recognized when preceded by a space (apparently Github's SCSS syntax highlighter doesn't recognize them either).

Test 1:

``` SCSS
.item {
  background: #{red} /* comment */;
  background: #{red}/* comment */;
}
```

Output (v3.4.19):

```
Error: Invalid CSS after "...ground: #{red}/": expected expression (e.g. 1px, bold), was "* comment *\/;"
        on line 3
```

Test 2:

``` SCSS
.item {
  background: #{red} // comment
  ;
  background: #{red}// comment
  ;
}
```

Output (v3.4.19):

```
.item {
  background: red;
  background: red//comment;
}
```

**Comment by chriseppstein** [comment]
Pretty sure this will be fixed by #1778.

Easy enough to work around tho, right?

**Comment by andrew-skybound** [comment]
Of course.
